lib/main/main.dart

```diff
@@ -49,14 +46,14 @@ class _MyHomePageState extends State<MyHomePage> {
   bool _showSwipeText = false;
   bool _isFetchingSharedPreferences = false;
 
+  GoogleSignInApi api = GoogleSignInApi();
+
   @override
   void initState() {
     // TODO: implement initState
     super.initState();
 
-    Firestore.instance.settings(
-      timestampsInSnapshotsEnabled: true,
-    );
+    api.setTimestampsSnapshots();
```

Comment:
What is timestamp snapshots? It's a very Firebase oriented term and UI should not know about this. See if we can do the initialization in constructor or change it as below:
```Suggestion
    api.initialize();
```